docs/deploy.md

```diff
@@ -1,38 +1,66 @@
 # Deploy models with Cog
 
-Cog containers are Docker containers that serve an HTTP server for running predictions on your model. You can deploy them anywhere that Docker containers run.
+Cog containers are Docker containers that serve an HTTP server 
```

Comment:
Looks like you're using [semantic line breaks](https://sembr.org/)? I like it! My only concern is making sure others know to follow this convention, otherwise you lose the diffing benefits

Reply:
Well, gotta practice what I [preach](https://github.com/sembr/specification?tab=readme-ov-file#about) 😇 

I'd be very happy to adopt this as an internal standard. For now, I'm using the lack of sembr as a signal for what could use a review.

Reply:
Whoa sembr never heard of it but I like it!